build.cake

```diff
@@ -313,48 +319,49 @@ Task("CreateImage")
         }
     });
 
-Task("PackageFramework")
-    .Description("Creates NuGet packages of the framework")
+#tool GitLink
```

Comment:
How about moving `#tool` up top with the other `#tool`? Or moving the NUnit.Console `#tool` down to the test target - so the two are consistent. (But my personal style preference would be the former. 😄)

Reply:
@nunit/framework-team Looks like we have two opposite personal preferences and no strong opinions here– should I move it to the top?

Reply:
Top feels right to me. Didn't see any competing comment.

Reply:
:+1: The opposite personal preference is showcased in the PR 😁